packages/ckb-cli/src/dashboard/index.ts

```diff
@@ -130,7 +130,7 @@ const dashboard = (core: Core) => {
     ]
     const { transactions: txs } = block
     if (txs.length) {
-      const newTxs: Transaction[] = txs.map(tx => [
+      const newTxs: Transaction[] = txs.map((tx): [string, string, string] => [
```

Comment:
How is this related to dev deps pinning?

Reply:
A [type error](https://travis-ci.com/nervosnetwork/ckb-sdk-js/jobs/229503053#L278) was thrown in the CI by pinning the dependencies. It seems that `tuple` and `string[]` are regarded as incompatible types now. So I upgraded the dev deps and types in the second commit.